pkgs/desktops/deepin/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, pkgs, config, libsForQt5 }:
+{ lib, pkgs, config, libsForQt5, qt6Packages }:
```

Comment:
Are not all upgraded to qt6?

Reply:
Only `dde-application-manager` use qt6 so far

There are currently some new projects that only support qt6, such as `dde-shell`, `treeland`, they will take the place of `dde-dock`, `deepin-kwin`. But still under early development

The latest version of dde-launchpad supports qt 6, but it need qt >= 6.7 for [this patch](https://codereview.qt-project.org/c/qt/qtdeclarative/+/414503), so I still use qt5 to build launchpad